TypeScript/package.json

```diff
@@ -8,18 +8,19 @@
   },
   "keywords": [
     "TDD",
-    "Mocha",
+    "Jest",
     "testing"
   ],
-  "author": "Adam Dolah",
+  "author": "Muhammad Ahsan Ayaz",
   "license": "ISC",
   "devDependencies": {
     "@types/chai": "^4.3.3",
-    "@types/jest": "^29.1.1",
+    "@types/jest": "^29.5.3",
     "@types/node": "^18.8.0",
+    "jest": "^29.6.3",
     "mocha": "^10.0.0",
```

Comment:
As we are no longer using Mocha we should remove this from the devDependencies? Same with the @types/chai?